packages/core/src/config/options.ts

```diff
@@ -50,11 +51,14 @@ export const buildOptions = ({
     generatedDir: "generated",
     ponderDir: ".ponder",
     logDir: ".ponder/logs",
-    telemetryDisabled: Boolean(process.env.PONDER_TELEMETRY_DISABLED),
+
     port: Number(process.env.PORT ?? 42069),
     maxHealthcheckDuration:
       configOptions?.maxHealthcheckDuration ?? railwayHealthcheckTimeout ?? 240,
 
+    telemetryUrl: "https://ponder.sh/api/telemetry",
```

Comment:
Added the telemetry URL to options. Not currently configurable, but nice to have it in one place.